doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -297,8 +297,8 @@ The requirements in this section are to be delivered in addition to those in [se
 | `req.gen.ost.01` | General | Openness | The Architecture **should** embrace open-based standards and technologies. |[ra2.crt.001](chapter04.md#44-container-runtimes)<br>[ra2.crt.002](chapter04.md#44-container-runtimes)<br>[ra2.ntw.002](chapter04.md#45-networking-solutions)<br>[ra2.ntw.006](chapter04.md#45-networking-solutions)<br>[ra2.ntw.007](chapter04.md#45-networking-solutions)|
 | `req.inf.com.01` | Infrastructure | Compute | The Architecture **must** provide compute resources for Pods.  |[ra2.k8s.004](chapter04.md#43-kubernetes)|
 | `req.inf.stg.01` | Infrastructure | Storage | The Architecture **must** support the ability for an operator to choose whether or not to deploy persistent storage for Pods. |[ra2.stg.004](chapter04.md#46-storage-components)|
-| `req.inf.ntw.01` | Infrastructure | Network | The Architecture **must** support network resiliency. | |
-| `req.inf.ntw.02` | Infrastructure | Network | The Architecture **must** be fully redundant. | |
+| `req.inf.ntw.01` | Infrastructure | Network | The Architecture **must** support network resiliency on the Kubernetes nodes, from the physical interfaces to the Host OS. | |
```

Comment:
why "from the physical interfaces to the Host OS"?

Reply:
agreed to remove the part after the comma: "The Architecture **must** support network resiliency on the Kubernetes nodes"

Reply:
noted that there is no spec in chapter 4 addressing this requirement - only number of NIC ports on physical host

Reply:
fixed in https://github.com/cntt-n/CNTT/commit/029f5771254eb2c054fc2599ddcfdf89f3b232ae , please review